solr/core/src/java/org/apache/solr/update/processor/DistributedZkUpdateProcessor.java

```diff
@@ -786,52 +784,7 @@ protected List<SolrCmdDistributor.Node> setupRequest(
         // that means I want to forward onto my replicas...
         // so get the replicas...
         forwardToLeader = false;
-        String leaderCoreNodeName = leaderReplica.getName();
```

Comment:
Was exactly the same as `getReplicaNodesForLeader`, except for the timeout (has been added as an argument of the method).